src/utils/SignerUtils.ts

```diff
@@ -121,9 +122,46 @@ async function getSecretSigner(): Promise<Signer> {
   }
 }
 
+export function getArweaveJWKSigner(keyType: "readonly" | "read-write"): ArweaveWalletJWKInterface {
+  // If the keytype is readonly, we should generate a read-only key
+  // on the fly and return it.
+  if (keyType === "readonly") {
+    // This is a dummy key. It is meant to fail if used.
```

Comment:
The existing wording seems contradictory to the comment above and implies (for me) that it's a temporary key to be replaced by something else.

```suggestion
    // This is a dummy key. It is meant to fail if used to write to Arweave.
```